qcut/.claude/skills/native-cli/reference-vimax.md

```
@@ -0,0 +1,132 @@
+---
+
+All `vimax:*` commands share these override flags:
+
+| Flag | Description |
+|------|-------------|
+| `--llm-model` | Override LLM agent model |
+| `--image-model` | Override image generation model |
+| `--video-model` | Override video generation model |
+| `--output-dir`, `-o` | Output directory |
+
+### `vimax:idea2video`
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)